modules/microshift-accessing-cluster-remotely.adoc

```diff
@@ -29,17 +29,17 @@ Use the following procedure to access the {product-title} cluster from a remote
 +
 [source,terminal,subs="attributes+"]
 ----
-[user@microshift]$ MICROSHIFT_MACHINE=<name or IP address of {product-title} machine>
+[user@workstation]$ MICROSHIFT_MACHINE=<name or IP address of {product-title} machine>
 ----
 
-. As `user@workstation`, copy the generated `kubeconfig` file from the {op-system} machine running {product-title} to your local machine by running the following command:
+. As `user@workstation`, copy the generated `kubeconfig` file that contains the host name or IP address you want to connect with from the {op-system} machine running {product-title} to your local machine by running the following command:
 +
 [source,terminal]
 ----
 [user@workstation]$ ssh <user>@$MICROSHIFT_MACHINE "sudo cat /var/lib/microshift/resources/kubeadmin/kubeconfig" > ~/.kube/config
```

Comment:
This command needs to point to the kubeconfig file with the info for accessing the cluster remotely. The file it refers to now is the one for local access.

```suggestion
[user@workstation]$ ssh <user>@$MICROSHIFT_MACHINE "sudo cat /var/lib/microshift/resources/kubeadmin/$MICROSHIFT_MACHINE/kubeconfig" > ~/.kube/config
```